### DISCUSSION TITLE
Consider a more UX focused stance on what Files allows users to do

### DISCUSSION THREAD

**Discussion by hanaral** [elementary/files#1605] [Ideas]
### Intro:

So it seems that the idea of messing with the way that the user home directory looks isn't very popular. 
However, since elementary's approach is UX first, you have to consider this from the perspective of someone who doesn't know, want to know or care what a _'leenooks'_ is.

### Le problem:

A new user right now has an almost an entire OS that is uncharted territory to them. They will quite literally latch on to anything that feels familiar _(a la people immediately searching where the minimise button is)_, which of course includes the file manager.
I don't particularly like file managers or the idea that 'modern computing' involves the digital equivalent of stacking shelves, but if it exists and is available to the user by default, so be it. 

**There are xserver config files stored in a user's home, so are their flatpak apps, runtimes, repos and cache.**

#### Prevent deletion of critical stuff by default _(muh dotfiles)_
macOS does not have any of the crucial components of it's system on the same partition (or capsule, whatever the new apfs thing is), let alone in the user's directory. 
Files could easily have a gsetting for whether to throw a messegedialogue telling them items meant for the OS are undeletable (for blacklisted files like .Xauthority and .local/share/flatpak/repo)

@cassidyjames I hope this place suits this kind of discussion better, you might want to pin things that gain tracktion for better visibility

**Comment by jeremypw** [comment]
I think the aim should be that a new non-technical user should not need to use the file-manager at all to accomplish the things they are interested in, tbh.

**Reply by hanaral** [reply]
Well, sandly the majority of people micromanage external storage on a regular basis, and until elementary creates a system that autotrashes the zettabytes of garbage the average user downloads and forgets about Files will be something they need.

**Reply by jeremypw** [reply]
Maybe we need an elementary app for safely freeing up disk space.  There a number of third-party apps available.

**Reply by hanaral** [reply]
I've been tossing around an idea in my head for a recovery dialogue in appcenter if it detects something being funky about the flatpak config files.
I could literally just say 'would you like to repair the issue?' and run `flatpak --user repair` with a progressbar (and detailed output in an expandable terminal style class). 
It might also be good to make it look like it has some kind of officiality to it being a system fixing thing so that users arent scared off.
